src/citrine/resources/table_config.py

```diff
@@ -22,8 +22,6 @@
 from citrine.gemtables.variables import Variable, IngredientIdentifierByProcessTemplateAndName, \
     IngredientQuantityByProcessAndName, IngredientQuantityDimension
 
-logger = getLogger(__name__)
-
```

Comment:
The oft-quoted distinction between logger and warnings is:
> warnings.warn() in library code if the issue is avoidable and the client application should be modified to eliminate the warning
> logging.warning() if there is nothing the client application can do about the situation, but the event should still be noted

Since the consumer of citrine-python should modify their code to eliminate the warning, I think we want to emit warnings.warn()